website/content/docs/secrets/transform/ff3-tweak-details.mdx

```diff
@@ -0,0 +1,77 @@
+# FF3-1 Tweak Usage Documentation
+
+## Introduction 
+
+The Vault Transform Secrets Engine uses the FF3-1 algorithm for FPE 
+(format preserving encryption) transformations. The key benefit of format
+preserving encryption schemes is that it allows for the secure encryption 
+of plaintexts over small domain spaces [1]. However, this comes with 
+the introduction of a parameter called the tweak, which is supplied to the 
+FF3-1 algorithm. Up until recently, the formal definitions of security in FPE 
+schemes have assumed that an attacker only has access to one tweak when probing 
+the codebook for a particular key. However, following the work of Bellare, this 
+notion was changed to assume that an attacker could control multiple tweaks when 
+generating a codebook and trying to create a nontrivial distinguisher for FF3-1. 
+
+This page will detail some best practices when using tweaks.
+
+Note that this guidance only holds when the tweak source selected when using the Vault
+FF3-1 transform is `supplied` or `internal`. 
+
+## FF3-1 Best Practices:
+
+Vault recommends the following practices when utilizing tweaks with the FF3-1 
+transform.
+
+* Encryption should be as restricted as possible via ACLs. 
+* Tweaks are not user controlled, but are randomly chosen and treated as 
+sensitive (if not secret). 
+* Tweaks make use of the max length allowed for the bytestring,
+and have high entropy (as uniformly variable as possible over the max tweak 
+domain space and differing with each encryption). Tweaks should utilize the
+full domain space.
+* Care should be taken to ensure that the same tweak is not used to encrypt
+two of the same values that correspond to different underlying entities.
+
+The following section aims to give some broad motivation for these recommendations. 
+For details, please see the Further Resources section at the bottom of the page.
+
+## FF3-1 Tweak Usage
+
+FF3-1 is a tweakable encryption scheme that was created to solve the problem of
+encrypting on small domains. Tweaks were introduced to synthetically increase the
+domain space. With the introduction of tweaks, an attacker must now have the
+correct encryption of the plaintext and the correct tweak in order to backtrack 
+via a codebook from an encryption to a plaintext. 
+
+However, this backfires when the number of tweaks used in FF3-1 setups
```

Comment:
s/backfires/can be risky/  In general go for more neutral phrasing.  Backfires might cause someone to assume immediate failure

Reply:
Definitely, good catch! Fixed!